include/openrave/kinbody.h

```diff
@@ -142,6 +142,9 @@ class OPENRAVE_API JointControlInfo_RobotController : public InfoBase
     std::string controllerType; ///< the type of the controller used to control this joint
     //int robotId = -1; ///< index of the robot within that controller?
     boost::array<int16_t, 3> robotControllerAxisIndex = {-1, -1, -1}; ///< indicates which DOF in the robot controller controls which joint axis (up to the DOF of the joint). -1 if not specified/not valid.
+    boost::array<dReal, 3> robotControllerAxisMult = {1.0, 1.0, 1.0}; ///< indicates the multiplier to convert the joint values from openrave to the robot controller, per joint axis.
+    boost::array<dReal, 3> robotControllerAxisOffset = {0.0, 0.0, 0.0}; ///< indicates the offset, in the robot controller units, that should be applied to the joint values, per joint axis.
+    boost::array<std::string, 3> robotControllerAxisProductCode = {"", "", ""}; ///< indicates the product code of the robot controller used to control each joint axis.
```

Comment:
@MiraiHattori @kanbouchou can you explain why you need a "productCode" per axis? What are some examples of product codes? thanks

Reply:
@rdiankov product code is an array of strings to identify which servo device the robot controller has to connect to, like a serial number of each device.
usually each axis is controlled by one servo controller device, so it's stored per axis.
it can be `{ "001", "002", ...}` or `{"Com001", "Com002", ...}`, depending on the user of the `robotControllerAxisProductCode` field.

Reply:
@MiraiHattori @kanbouchou How is that different from "ElectricMotorActuatorInfo::model_type"?

Reply:
@MiraiHattori Got it, please also add this to the comment since the current comment is confusing. thanks

Reply:
@rdiankov Sure, I added comments about it. I also updated the comment of `robotControllerAxisMult` about units.